### ISSUE TITLE
Issue with `async let _ = …` for void type

### ISSUE THREAD

**Issue by paxos** [nicklockwood/SwiftFormat#1255]
Example:

```
async let _ = try await funcReturningVoid()
```

This snippet compiles without a warning on Xcode 13.4

Is getting formatted to:

```
async _ = try await funcReturningVoid()
```

which is not compiling anymore.

Is this a bug or is there a configuration value for this somewhere?

**Comment by nicklockwood** [comment]
This is a bug - thanks for flagging it

**Comment by nicklockwood** [comment]
This is caused by the `redundantLet` rule, so disabling that should be a workaround for now.

**Comment by paxos** [comment]
Thanks!

Also, fantastic project: I could not do my work without this anymore <3

**Comment by nicklockwood** [comment]
@paxos Fixed in 0.49.18

**Comment by paxos** [comment]
Thanks, thats amazing 🚀
